docs/conf.py

```diff
@@ -22,10 +23,23 @@
 
 from string import Template
 
+on_rtd = os.environ.get('READTHEDOCS') == 'True'
+
 
 def get_tutorial_images():
     TUTORIAL_URL = "https://fz-juelich.sciebo.de/s/KcF29hPNRzkxN6q/download"
-    TUTORIAL_DIR = 'videos/mp4'
+    if on_rtd:
+        # Processing the docs on RTFD will use the videos static path to copy the videos to _images
+        # and the video url gets changed to that location.
+        TUTORIAL_DIR = 'videos/mp4'
+    else:
+        # Locally the origin path is used to define the url.
+        # That means we need to store the videos on that location
+        if not os.path.exists("../docs/_build"):
+            logging.debug("Build directory not found.")
+            return
+        TUTORIAL_DIR = '../docs/_build/html/videos/mp4'
```

Comment:
How does this interact with building the docs for a target different from html?

Reply:
I don't really understand the issue yet, but treating RTD and non-RTD differently and directly writing to the `_build` directory here seems wrong.

Reply:
so true, I spend yesterday a lot time trying, to get only to this workaround. 

Ha, and now I have a new idea what got wrong.

Reply:
hmm, no, no typo in the extensions definition. It just makes only things now different on RTFD and local.

Reply:
See https://github.com/Open-MSS/MSS/pull/2458#issuecomment-2288591083. The underlying issue is a version mismatch of sphincontrib-video between conda-forge (local) and PyPI (RTD).